homeassistant/components/hunterdouglas_powerview/sensor.py

```diff
@@ -39,6 +39,7 @@ class PowerviewSensorDescriptionMixin:
     update_fn: Callable[[BaseShade], Any]
     native_value_fn: Callable[[BaseShade], int]
     create_sensor_fn: Callable[[BaseShade], bool]
+    visible_default: bool
```

Comment:
We already have the entity description attribute `entity_registry_visible_default` in the base class.

Reply:
well that cuts the PR down 😄 

1 line change submitted with the suggestion to disable and not hide